maintainers/maintainer-list.nix

```diff
@@ -2617,6 +2617,12 @@
     githubId = 841969;
     name = "David Kuo";
   };
+  drzoidberg = {
```

Comment:
could you please sort your entry alphabetically?

Reply:
I will.